indico/modules/rb/views/user/rooms.py

```diff
@@ -156,7 +156,9 @@ class WPRoomBookingRoomStats(WPRoomBookingBase):
     def _getBody(self, params):
         params['period_options'] = [
             ('pastmonth', _('Past month')),
+            (params['last_month'], _('Last month')),
```

Comment:
s/Last/Previous/? I think that would make it clearer that it's using calendar months and not "today-30d"